FIX address sender mailer config (#1549)
shopware/shopware: changelog/_unreleased/2020-12-03-Fix-custom-sender-email.md

---
title:          Fix custom sender email address
issue:
author:         Timothee Montias
author_email:   timothee.motnias@epita.fr
---
# Core

>     * Cahnged the MailService class by changing the senderEmail variable affectation begining with the custom email address.

Reviewer: ```suggestion
* Changed the `senderEmail` variable affectation beginning with the custom email address in the `Shopware\Core\Content\MailTemplate\Service\MailService`
```
